app/views/paymentReport.html

```diff
@@ -0,0 +1,165 @@
+    <div class="col-xs-2 details-heading" translate>
+      Conference Long Name:
+    </div>
+    <div class="col-xs-3 details-heading">
+      {{report.conferenceLongName}}
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="col-xs-2 details-heading" translate>
+      Conference ID:
+    </div>
+    <div class="col-xs-3 details-heading">
+      {{report.conferenceId}}
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="col-xs-2 details-heading" translate>
+      Conference Contact:
+    </div>
+    <div class="col-xs-3 details-heading">
+      {{report.contactPersonName}}
+    </div>
+  </div>
+  <div class="row form-group">
+    <div class="col-md-2 details-heading">
+      <label translate for="report-select-id">Report creation date: </label>
+    </div>
+    <div class="col-xs-3 details-heading">
+      <select
+        ng-model="queryParameters.currentReportId"
+        ng-options="r.id as r.createdTimestamp for r in reports"
+        class="form-control"
+        id="report-select-id"
+      >
+        <option value="">New Report</option>
+      </select>
+    </div>
+
+    <div class="row form-group">
+      <div class="col-xs-12 table-responsive">
+        <table class="table">
+          <thead>
+            <tr>
+              <th>
+                <a href="" translate>Transaction Id</a>
+              </th>
+              <th>
+                <a href="" translate>First Name</a>
+              </th>
+              <th>
+                <a href="" translate>Last Name</a>
+              </th>
+              <th>
+                <a href="" translate>Check Number</a>
+              </th>
+              <th>
+                <a href="" translate>Amount</a>
+              </th>
+              <th>
+                <a href="" translate>Include into report</a>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              ng-repeat-start="r in report.paymentReportEntries"
+              ng-class="{'active': $even}"
+              class="noselect"
+            >
+              <td>
+                {{r.transactionId}}
+              </td>
+              <td>
+                {{r.firstName}}
+              </td>
+              <td>
+                {{r.lastName}}
+              </td>
+              <td>
+                {{r.checkNumber}}
+              </td>
+              <td>
+                {{(r.checkNumber == "REFUND" ? "(" : "") + (r.amount |
+                localizedCurrency:conference.currency.currencyCode) +
+                (r.checkNumber == "REFUND" ? ")" : "")}}
+              </td>
+              <td>
+                <input
+                  type="checkbox"
+                  ng-model="r.included"
+                  ng-change="refreshExcludedIds(r.paymentId)"
+                  ng-disabled="queryParameters.currentReportId"
+                />
+              </td>
+            </tr>
+            <tr ng-repeat-end></tr>
```

Comment:
I guess part of my question was whether you need the `<tr ng-repeat-end></tr>` at all? You could just use `ng-repeat` above instead of `ng-repeat-start` if that's the case.